### ISSUE TITLE
error: no such module 'Darwin'

### ISSUE THREAD

**Issue by Its-Kenta** [Lancelotbronner/swift-raylib#2]
Compilation checks are required to ensure Darwin is not imported outside of Apple ecosystem. Getting the following spam on Linux:

```
swift-raylib/Sources/Raylib/Math/Integration/Double.swift:8:8: error: no such module 'Darwin'
```
Same will most likely appear on Windows. 
This is also present in `Float.swift` and `Vector FloatingPoint.swift`

**Comment by Lancelotbronner** [comment]
The math module definitely needs a review, I'll keep cross-platform in mind. I'd like to setup some VMs to test properly on all 3 platforms.

**Comment by Its-Kenta** [comment]
If you need any testing on Linux do let me know and I can help out 👍

**Comment by Lancelotbronner** [comment]
The latest commit should have fixed this issue, along with building 5.0 support. I'll be working on adding missing content and fixing examples now!
